Lang/zh-s.json

```diff
@@ -0,0 +1,126 @@
+{
+	"lang": "简体中文 (ZH-S)",
+	"variables": {
+		"labelEnableHack": "启用",
+		"emuStatusRunning": "模拟器运行中",
+		"logWindowTitle": "Executando fpPS4",
```

Comment:
Thanks @nini22P, awesome job!
It's only missing this label - after this I can merge it to `main`.

Reply:
Thanks, it's fixed

Reply:
Awesome job!
Do you think it needs any more changes? - I'm ready to merge.

Reply:
No need to change.